akka-remote/src/main/scala/akka/remote/artery/LruBoundedCache.scala

```diff
@@ -3,39 +3,19 @@
  */
 package akka.remote.artery
 
-import akka.util.{ OptionVal, Unsafe }
+import akka.util.Unsafe
 
 import scala.annotation.tailrec
 import scala.reflect.ClassTag
 
 /**
  * INTERNAL API
  */
+@deprecated(message = "Use akka.util.Unsafe instead", since = "2.5.5")
 private[remote] object FastHash {
```

Comment:
this is internal API, so we can just remove it, and change all calls to `Unsafe.fastHash`

Reply:
Will remove, I left it because binary compatibility broke.